8 Ball/Modules/Prediction/PredictionViewController.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  ViewController.swift
+//  8 Ball
+//
+//  Created by Roman Topchii on 13.01.2022.
+//
+
+import UIKit
+
+protocol QuestionListener {
+    func refreshQuestion(_ question: String?)
+}
+
+protocol PredictionPresenter {
+    func presentPrediction()
+}
+
+class PredictionViewController: UIViewController {
+    
+    var model: PredictionService!
```

Comment:
Может переменную service назвать, а то как-то странно model тут выглядит)?
```suggestion
    private let model: PredictionService
```